### PR TITLE
Add example programs for log creating and viewing 

### PR DESCRIPTION
**Pull request by Goala**

### CODE REVIEW

File: src/zale_export_log_bal.prog.abap [hunk 1]
```diff
@@ -11,10 +11,14 @@ PARAMETERS suobject TYPE balhdr-subobject OBLIGATORY.
 START-OF-SELECTION.
 
   DATA filter_values TYPE zif_ale_log_reader=>ty_filter_values.
+  DATA logs TYPE zif_ale_log_reader=>ty_logs.
 
   filter_values = VALUE #( ( key = 'OBJECT' value = object )
                            ( key = 'SUBOBJECT' value = suobject ) ).
 
+  logs = NEW zcl_ale_log_reader_bal(  )->zif_ale_log_reader~read( filter_values ).
```
Comment (JohannesKonings) [inline src/zale_export_log_bal.prog.abap]: I would recommend a inline declaration like data(logs) = NEW ...

File: src/zale_export_log_bal.prog.abap [hunk 2]
```diff
@@ -11,10 +11,14 @@ PARAMETERS suobject TYPE balhdr-subobject OBLIGATORY.
...
+
+  cl_demo_output=>display( logs ).
```
Comment (JohannesKonings) [inline src/zale_export_log_bal.prog.abap:20]: should be replaced with the further implementation of the converter und connector

**Review by JohannesKonings — COMMENTED** [review]
@Goala is the intention of this pull request a program for creating example logs and and a little cleanup?
If you want you can add some change recommended in the comments, if not ich can merge this pull request :)

File: src/zale_export_log_bal.prog.abap (hunk 2 above)
Comment (Goala) [reply src/zale_export_log_bal.prog.abap:20]: Changed it.

File: src/zale_export_log_bal.prog.abap (hunk 1 above)
Comment (Goala) [reply src/zale_export_log_bal.prog.abap]: Changed it.

File: src/zale_export_log_bal.prog.abap (hunk 2 above)
Comment (Goala) [reply src/zale_export_log_bal.prog.abap:20]: already inside this branch? 
this was only a temporary solution to display the logs and for using the reader class.

File: src/zale_export_log_bal.prog.abap [hunk 3]
```diff
@@ -15,6 +15,9 @@ START-OF-SELECTION.
   filter_values = VALUE #( ( key = 'OBJECT' value = object )
                            ( key = 'SUBOBJECT' value = suobject ) ).
 
+  DATA(logs) = NEW zcl_ale_log_reader_bal(  )->zif_ale_log_reader~read( filter_values ).
```
Comment (Goala) [inline src/zale_export_log_bal.prog.abap:18]: @JohannesKonings

**Review by Goala — COMMENTED** [review]
@JohannesKonings yes the intention was the example log program and displaying them :)

File: src/zale_export_log_bal.prog.abap (hunk 2 above)
Comment (JohannesKonings) [reply src/zale_export_log_bal.prog.abap:20]: in one of the next steps is OK :)

**Review by JohannesKonings — APPROVED** [review]
LGTM

**Pull request merged**